roles/openshift_logging/tasks/oc_secret.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- command: >
```

Comment:
@kwoodson do we have a timeline on when the oc_secret module will be ported? I didn't have access to the trello card where this work is being tracked...